src/deployment/flavors.md

```diff
@@ -3,41 +3,158 @@ title: Creating flavors for Flutter
+2. In the Build Settings, set the Product Name value to match each flavor.        
+![Step 2 using flavors image.](/assets/images/docs/flavors/step2-using-flavors.png){:width="100%"}             
+3. Add the display name to info.plist.        
+Create a new key in info.plist called **Bundle Display Name** 
+with the value `$(PRODUCT_NAME)`. 
+![Step 2 using flavors image.](/assets/images/docs/flavors/step2-using-flavors.png){:width="100%"}    
+
+Now you have set up your development flavor by making a `development` scheme 
+in Xcode and setting the build configurations for that scheme. 
+
+For more information, skip to the [“Launching your app flavors”][] 
+section at the end of this document.
+
+## Using flavors in Android
+
+Setting up flavors in Android can be done in your projects 
+**build.gradle** file.
+
+1. In your IDE, inside your Flutter project, 
+navigate to **android** > **app** > **build.gradle**.   
+
+![Step 1 using flavors in Android image.](/assets/images/docs/flavors/android-step1.png){:width="100%"}    
+
+Add a flavors object with the specified environments along with values for 
+dimension, resValue, and applicationId. 
+The name of the application for each build is located in resValue 
+(as shown in the screenshot below from VSCode).
+## Setting up launch configurations
+
+If you are using VS Code as your IDE, the following steps 
+show you how to set up launch configurations. 
+Adding a **launch.json** file allows you to run the terminal command 
+`flutter run --flavor [environment name]`. 
+1. In the root directory of your project, add a folder called **.vscode**.    
+2. Inside the **.vscode** folder, create a file named **launch.json**.    
+3. In the **launch.json** file, add a configuration object for each flavor. 
+Each configuration has a **name**, **request**, **type**, **program**, 
+and **args** key.
+
+![Image for VSCode launch configuration step.](/assets/images/docs/flavors/vscode-config.png){:width="100%"}  
+
+## Launching your app flavors
+
+Once you’ve set up your flavors in iOS and/or Android, 
+make modifications to your Dart code in **lib** > **main.dart** 
+that allow you to consume the flavors. 
+Run `flutter run --flavor [environment name]` in your terminal. 
+Check out the [Flutter repo][] for integration test examples of 
+build flavors for [iOS(Xcode)][] and [Android][]. 
+
+
+With this document you were able to: 
+* Set up flavors in iOS by creating a scheme and 
+modifying the build configurations. 
+* Set up flavors in Android by modifying the **build.gradle** file.
+* Set up launch configurations for VSCode.
+* See examples of flavor use in production apps. 
+
+## More information
+
+* [Build flavors in Flutter (Android and iOS) with different Firebase projects per flavor Flutter Ready to Go][]
+* [Flavoring Flutter Applications (Android & iOS)][]
+* [Flutter Flavors Setup with multiple Firebase Environments using FlutterFire and Very Good CLI][]
```

Comment:
For these massively long links, I'd set up a shortcut. It's also more bulletproof if someone modifies the article name at all. (It happens) You can do that like this:

```suggestion
* [Build flavors in Flutter (Android and iOS) with different Firebase projects per flavor Flutter Ready to Go][article1]
* [Flavoring Flutter Applications (Android & iOS)][article2]
* [Flutter Flavors Setup with multiple Firebase Environments using FlutterFire and Very Good CLI][article3]

[article1]: {{site.medium}}/@animeshjain/build-flavors-in-flutter-android-and-ios-with-different-firebase-projects-per-flavor-27c5c5dac10b
[article2]: {{site.medium}}/flutter-community/flavoring-flutter-applications-android-ios-ea39d3155346
[article3: https://codewithandrea.com/articles/flutter-flavors-for-firebase-apps/
```

Also, are all of these articles free? We generally avoid recommending any articles behind a paywall.